src/evict/evict_page.c

```diff
@@ -113,7 +113,7 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
     force_evict_hs = false;
     local_gen = false;
 
-    __wt_verbose(
+    __wt_verbose_debug3(
```

Comment:
I prefer that let this be, because knowing the page is quite important info

Reply:
Thanks, fixed in ce6c52b.